gateway/modules/crud/sql/read.go

```
@@ -64,6 +65,9 @@ func (s *SQL) generateReadQuery(ctx context.Context, col string, req *model.Read
 			}
 		}
 		if req.Options.Skip != nil {
+			if model.DBType(s.dbType) == model.SQLServer && len(req.Options.Sort) == 0 {
```

Let the database throw an error